ci/azure/posix.yml

```diff
@@ -90,3 +90,8 @@ jobs:
           Write-Error "$($matches[1]) tests failed"
         }
       displayName: Check for test failures
+    - script: |
+        export PATH=$HOME/miniconda3/bin:$PATH
+        source activate pandas-dev
+        python ci/print_skipped.py
+      displayName: 'print skipped test'
```

Comment:
Can you capitalize the `p` and use `tests` instead of `test`? so the display name is `Print skipped tests`